static/app/components/assigneeSelectorDropdown.tsx

```diff
@@ -437,7 +526,7 @@ export class AssigneeSelectorDropdown extends Component<
       }
     );
 
-    return suggestedAssignees.filter(owner => !!owner) as SuggestedAssignee[];
+    return suggestedAssignees.filter((owner): owner is SuggestedAssignee => !!owner);
```

Comment:
the type is `SuggestedAssignee | null` and i'm removing the null items, the `is SuggestedAssignee` tells typescript what i'm checking for otherwise it kinda doesn't know

Reply:
its the more correct way of doing as SuggestedAssignee[] which is removing the null type via casting